src/core/src/pass/sdpa_to_paged_attention.cpp

```diff
@@ -30,13 +30,10 @@ static std::shared_ptr<v0::Parameter> setName(std::shared_ptr<v0::Parameter> nod
 
 bool ov::pass::SDPAToPagedAttention::run_on_model(const std::shared_ptr<ov::Model>& model) {
     RUN_ON_MODEL_SCOPE(SDPAToPagedAttention);
-
-    for (auto& op: model->get_ordered_ops()) {
-        if (std::dynamic_pointer_cast<ov::op::v13::ScaledDotProductAttention>(op)) {
-            break;
-        }
-
-        std::cout << "No ScaledDotProductAttention operation observed in the graph, cannot perform the SDPAToPagedAttention transformation." << std::endl;
+    if (!ov::op::util::has_op_with_type<ov::op::v13::ScaledDotProductAttention>(model)) {
+        std::cout << "No ScaledDotProductAttention operation observed in the graph, cannot perform the "
```

Comment:
Did we agree to add some asserts to these Transformations?
should it be also the assert?